test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Metric/MetricTests.cs

```diff
@@ -98,5 +113,36 @@ private async Task ExecuteAndValidate<T>(Func<Task<T>> action, params (Type type
                 }
             }
         }
+
+        private async Task ExecuteAndValidateTransaction<T>(Func<Task<T>> action, params (Type type, int count, string resourceType)[] expectedNotifications)
```

Comment:
If this method is specific to transaction metrics and not required to be reused, how about combining in a single method?